### ISSUE TITLE
Issues with Invisible Button Directive

### ISSUE THREAD

**Issue by simongh** [leNicDev/ng-hcaptcha#9]
There appears to be some issues with the invisible button directive.

The simplest, is there's a typo `cancelBuble` instead of `cancelBubble`.

More complex is I can't get the events to fire. I have this:
 `<button ngHcaptchaInvisibleButton (click)="next($event)" (verify)="onVerify($event)"`

The verify event never fires. My click handler does though. Using the debugger in Chrome, The click handler fires in the directive, followed by my handler in my component and then that's it.

When I fiddled with the code & commented out the 4 lines in the click handler to stop event propagation, it started raising the verify event. It looks like one of these stops hcaptcha raising events.
```
 event.stopPropagation();
 event.preventDefault();
 event.cancelBubble = true;
 event.stopImmediatePropagation();
```
I can start a pull request for this, but I'm not sure why it's doing it to start with, or how to get around it.

**Comment by simongh** [comment]
After further testing, it's related to the `stopImmediatePropagation`

There does appear to be an open Angular issue regarding this method & multiple event handlers

**Comment by leNicDev** [comment]
Hi @simongh thanks for reporting this issue.
One way to bypass this issue would be to rename the event from `click` to something else. I initially decided to use the native click event to prevent confusion.

**Comment by simongh** [comment]
Also seem to be missing the tsconfig.base.json, which makes it tricky to build.

I'm not so fussed about the click event, since I can move everything that happened in the click handler into the onverify handler. 

I'm trying to hack something together to work around the issue by removing the `stopImmediatePropagation` - without that nothing works. If I can get around that, then I can update back to this once you've got a fix in place.

**Comment by simongh** [comment]
Although, actually I have some validation that would be nice to trigger in the click before firing the hcaptcha execute. Then the hcaptcha would only run when the validation passed.

**Comment by leNicDev** [comment]
@simongh Thanks a lot for your contribution! It might be a good idea to deprecate and remove the click handler in the near future.  I will run a few tests this week to release v1.0.0-beta.3 as soon as possible.

**Comment by simongh** [comment]
No problem. Happy to help. Thanks for responding so quickly

**Comment by leNicDev** [comment]
Has been released as 1.0.0-beta.3. Thanks a lot for your contribution!
